README.md

```diff
@@ -95,3 +95,68 @@ You can also specify multiple namespaces when using the hook:
 ```
   const t = useTranslation(['common', 'distribution']);
 ```
+
+## Android App
+
+### Build the remote server lib
+
+The Android app needs the remote server build as a shared lib.
+
+Make sure the Android NDK is installed and the env var `NDK_HOME` is set, for example to `~Android/Sdk/ndk/22.1.7171670`.
+Currently the build script requires the Android API 26, make sure this version is installed.
+
+The remote server source code should be located in the `../remote-server` directory.
+If not clone it by doing:
+
+```
+cd ..
+git clone git@github.com:openmsupply/remote-server.git
+```
+
+Unsolved issue that will hopefully be fixed at some point:
+
+1. in `../remote-server/Cargo.toml` add the "vendored" feature to openssl by changing the line:
+
+   `openssl = { version = "0.10", features = ["v110"] }`
+
+   to:
+
+   `openssl = { version = "0.10", features = ["v110", "vendored"] }`
+
+2. The x86 target seem to be broken and crashes when loading the server lib
+
+To build the remote server lib and copy it to the Android project run:
+
+```
+npm run android:build-remote_server
```

Comment:
We're using yarn elsewhere in preference to npm, so it might be best to avoid confusion
```suggestion
yarn android:build-remote_server
```